### ISSUE TITLE
Simple Key Loader - Feature Request

### ISSUE THREAD

**Issue by joeyMjTaylor** [IDI-Systems/acre2#563]
Feature Request: 
The Simple Key Loader, AKA SKL, Officially known as the AN/PYQ-10(C). This little device right here is used to upload COMSEC and encrypted keys to a radio for Operation Security. The SKL can have keys and times to have the frequency hop feature (FH). FH is used to ensure communications are secured and enemies can not access the friendly forces radio traffic. I noticed while playing with the 152's there was a COMSEC option. Though not sure if fully implemented, this would be a good feature for those MilSim's with some sort of Communication's Shop.

I was a Commo Specialist when I was in the Army, so any questions please feel free to let me know!

![simple-key-loader-skl-ppt-of-army-skl-powerpoint-class-1](https://user-images.githubusercontent.com/41753670/43275550-1ddbd84c-90d0-11e8-983a-30581b2ef586.jpg)
![simple-key-loader-skl-army-fielding-strategy-system-overview-of-army-skl-powerpoint-class-3](https://user-images.githubusercontent.com/41753670/43275551-20b248da-90d0-11e8-9db0-d519d954d142.jpg)

**Comment by TheMagnetar** [comment]
Yeah, COMSEC should receive an overhaul. However, for this to be implemented we would need some detailed  info on the device itself.
